### PR TITLE
[fix] Fixing COCO converter to include all images from dataset

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

**Review by kielnino — CHANGES_REQUESTED** [review]
The proposed changes don't go far enough, because the `labels` list would be empty for images without annotation, so the images are kicked out again on `continue` in line 447.

I think we need to go much further and add a "no annotation" class to the labels, so that the images without annotation remain in the list.

**Comment by lfeldb** [comment]
@kielnino would my pr be a solution for you?

#47

**Comment by kielnino** [comment]
@lfeldb Yes I think this would work.

File: label_studio_converter/converter.py [hunk 1]
```diff
@@ -467,31 +484,11 @@ def convert_to_coco(self, input_data, output_dir, output_image_dir=None, is_dir=
...
-                    images.append({
-                        'width': width,
-                        'height': height,
-                        'id': image_id,
-                        'file_name': image_path
-                    })
-                    first = False
+                if 'original_width' not in label or 'original_height' not in label:
+                    logger.warning(f'original_width or original_height not found in {image_path}')
+                    continue
```
Comment (makseq) [inline label_studio_converter/converter.py:489]: As I understand this continue will skip all image that don't have 'original_width' in annotations. However we wanted to include all images from dataset.

**Review by makseq — CHANGES_REQUESTED** [review]

File: label_studio_converter/converter.py [hunk 2]
```diff
@@ -467,31 +484,11 @@ def convert_to_coco(self, input_data, output_dir, output_image_dir=None, is_dir=
                     logger.warning("Unknown label type: " + str(label))
                     continue
 
-                # get image sizes
-                if first:
```
[the remark this answers is not in the document]
Comment (KonstantinKorotaev) [reply label_studio_converter/converter.py:471]: Fixed

File: label_studio_converter/converter.py (hunk 1 above)
Comment (KonstantinKorotaev) [reply label_studio_converter/converter.py]: Fixed

**Review by makseq — APPROVED** [review]

**Pull request merged**